hooks/pre-commit

```diff
@@ -6,7 +6,7 @@ else
     VERSION=$(python3 -c 'import sys; v = sys.version_info; print("{}.{}".format(v[0], v[1]));')
 fi
 python3 -m flake8 -j auto
-if [ $VERSION = '3.4' ]; then
+if [ $VERSION != '3.3' ]; then
```

Comment:
why don't you use `$VERSION = '3.4' or $VESRION = '3.5'`? I am afraid lower version of python might pass this `if`.